project.yaml

```diff
@@ -742,6 +745,7 @@ Layer:
                 foot,
                 bicycle,
                 tracktype,
+                CASE WHEN surface IN ('unpaved', 'compacted', 'dirt', 'earth', 'fine_gravel', 'grass', 'grass_paver', 'gravel', 'ground', 'mud', 'pebblestone', 'salt', 'sand', 'woodchips', 'clay') THEN 'unpaved' ELSE 'paved' END AS surface,
```

Comment:
Done in https://github.com/matkoniecz/openstreetmap-carto/commit/898339d4c7137efcdbed7f60c312f280fff1f66b

See also https://github.com/gravitystorm/openstreetmap-carto/commit/cc17b038055aeab387af3e374abc953e0bb48f99 that split overly long lines.